### ISSUE TITLE
Responsive behavior

### ISSUE THREAD

**Issue by tuecke** [globusonline/google-drive-registration-app#15]
When I shrink the width of the screen, the gdrive app pages try to maintain substantial left and right margins until it gets down to a very narrow screen.  It should drop the margins much sooner.

Also, as the screen width shrinks the "Google Drive “Globus Google Drive” successfully connected.\/" wraps a little funny, as the down arrow wraps onto its own line, which looks strange.

**Comment by corpulentcoffee** [comment]
Can you look at this, @teresas?
Ping me if you need help running the application locally, but `npm install && ember server` should be all you need after you pull the latest.

**Comment by teresas** [comment]
I'll take a look at it.

**Comment by teresas** [comment]
@corpulentcoffee I made changes and pushed in a branch teresa/responsive. Take a look and let me know if it needs adjustments or something Steve was looking for wasn't addressed.

**Comment by corpulentcoffee** [comment]
Thanks, I've merged these changes.

@tuecke If you have additional margin/padding adjustments, please re-open this with the specific page/URL you want them changed on (@teresa's changes in 2c6ffb296dfa3523456f75564d3294e889f0623f are fairly layout-specific) and let her know specific viewport sizes that are an issue (if you have the Developer Tools window open when you resize a Chrome window, it will display the viewport dimensions in its top-right corner).

**Comment by tuecke** [comment]
I'm nit picking, but this is what I think should happen...

As I reduce the window width, there seems to be various threshold widths at which the left margin jumps back up.  This seems odd to me.  I understand over a certain size that it no longer continues to widen, and instead everything centers, resulting in the left margin increasing.  But I would expect that after the window width decreases below a certain width, we then fix the left margin, and use everything else for input fields.

The first threshold is 1199px.  Can't we just keep the left margin fixed, and reduce the width of the input fields continuously as the window continues to shrink?  Also, "Endpoint Display Name" field header wraps to 2 lines, which does not look as good.  Can we leave a little more room for the field headers throughout this range so they don't wrap?

The next threshold is at 991px width, everything jumps and there is a big left and right margin, much smaller input boxes, and several more field headers wrap.  Can we shrink the left margin to essentially nothing (left justify the logo in the "Connect Your Google Drive to Globus" header), leave enough room for field headers to not wrap, and then adjust the input box widths to use the rest of the screen width?

The next jump is at 767px width, at which point the field headers move to be above the fields, rather than to the left of them, which makes sense.  But once again it adds back left and right margin, which does not make sense.  Once we get below this width it seems like we should use the whole screen width for the fields, like it does when we get below 575px.  That is, why not have the behavior that is currently below 575px extend out to 767px?

In summary, I think there should be 3 ranges with distinct behaviors:
1) Below 767px have the same behavior as it currently has below 575px
2) Between 768px and 1199px, have essentially very little left margin, keep enough room for field headers, and adjust the input boxes for the remaining width of the screen.
3) Above 1200px, keep the current behavior of fixing the content width, and centering it.

**Comment by teresas** [comment]
@tuecke I'm a bit concerned about getting too far from consistency with globus drupal and our other sites when it comes to the margins. In terms of narrowing the margins, we rely on Bootstrap for the breakpoints and our content containers, and we also had to add a bit of a margin on the left to accommodate the heading icons (so the text wouldn't wrap around them and look weird).

It might be worth taking some time to discuss the margins and such on Monday with @vytas to get his input and clarify how we want to adjust the layout.

**Comment by tuecke** [comment]
OK, I'll talk with @vytas on Monday.  I am approaching this from a position of what I think would be most aesthetically pleasing and useful, with complete ignorance about how it works, what are best practices, and how hard what I'm asking might be.  But I would like to understand why what I'm suggesting is either misguided (i.e., shouldn't be done), or technically difficult (i.e., can't be done with reasonable effort).
